src/Dapr.PluggableComponents.Tests/Adaptors/InputBindingAdaptorTests.cs

```diff
@@ -115,31 +109,30 @@ public async Task ReadClientCanceled()
         var reader = new AsyncStreamReader<ReadRequest>();
 
         fixture.MockComponent
-            .Setup(component => component.ReadAsync(It.IsAny<MessageDeliveryHandler<InputBindingReadRequest, InputBindingReadResponse>>(), It.IsAny<CancellationToken>()))
-            .Returns<MessageDeliveryHandler<InputBindingReadRequest, InputBindingReadResponse>, CancellationToken>(
-                (deliveryHandler, cancellationToken) =>
+            .ReadAsync(Arg.Any<MessageDeliveryHandler<InputBindingReadRequest, InputBindingReadResponse>>(), Arg.Any<CancellationToken>())
+            .Returns(
+                callInfo =>
                 {
-                    return Task.Delay(-1, cancellationToken);
+                    return Task.Delay(-1, (CancellationToken)callInfo[1]);
                 });
```

Comment:
There is no way do deconstruct callInfo so we don't need the `callinfo[1]` bellow, right?

Reply:
Checked over chat, this isn't a C# thing :(

Reply:
Would *love* that, and miss it from TypeScript, but not for lambdas in C#.